stdlib/3/types.pyi

```diff
@@ -200,6 +230,8 @@ class MemberDescriptorType:
     def __delete__(self, obj: Any) -> None: ...
 
 def new_class(name: str, bases: Tuple[type, ...] = ..., kwds: Dict[str, Any] = ..., exec_body: Callable[[Dict[str, Any]], None] = ...) -> type: ...
+if sys.version_info >= (3, 7):
+    def resolve_bases(bases: Tuple[type, ...]) -> Tuple[type, ...]: ...
```

Comment:
`bases` is not necessarily a tuple; it can be any iterable (https://github.com/python/cpython/blob/a9f05d69ccbf3c75cdd604c25094282697789a62/Lib/types.py#L79). Also, the entries in the tuple aren't necessarily types, because the point of this function is to replace non-types with an `__mro_entries__` method with types. I think it should just be `(Iterable[object]) -> Tuple[Any, ...]:`.

Reply:
Makes sense, though in that case I believe `new_class` should also be updated, as it is directly passing its `bases` argument to `resolve_bases`. Shall I update that signature as well, given version being 3.7?